vector/src/main/res/values/strings.xml

```diff
@@ -1049,6 +1049,11 @@
     <string name="media_saving_period_1_month">1 month</string>
     <string name="media_saving_period_forever">Forever</string>
 
+    <!-- Presence settings -->
+    <string name="settings_presence">Presence</string>
+    <string name="settings_presence_offline_mode">Offline mode</string>
+    <string name="settings_presence_offline_mode_summary">If enabled, this client won\'t send presence online status on sync</string>
```

Comment:
It's a bit technical for the users.
I would rephrase to "If enabled, you will always appear offline to other users, even when using the application."